src/components/Tablet/Tablet.tsx

```diff
@@ -0,0 +1,60 @@
+import {useRef} from 'react';
+import cn from 'bem-cn-lite';
+
+import type {TTabletStateInfo} from '../../types/api/tablet';
+
+import {InternalLink} from '../InternalLink';
+
+import {showTooltip} from '../../store/reducers/tooltip';
```

Comment:
oof, don't really like importing a module just for its type
creates an excess edge in dependency graph
but i think this comment is just to bring attention to it, not to fix right away